packages/loot-core/package.json

```diff
@@ -4,14 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build:node": "cross-env NODE_ENV=production webpack --config ./webpack/webpack.desktop.config.js",
-    "watch:node": "cross-env NODE_ENV=development webpack --config ./webpack/webpack.desktop.config.js --watch",
-    "build:api": "cross-env NODE_ENV=development webpack --config ./webpack/webpack.api.config.js; ./bin/copy-migrations ../api",
+    "build:api": "cross-env NODE_ENV=development ./bin/build-api",
     "build:browser": "cross-env NODE_ENV=production ./bin/build-browser",
     "watch:browser": "cross-env NODE_ENV=development ./bin/build-browser",
     "lint": "eslint .",
     "test": "npm-run-all -cp 'test:*'",
-    "test:node": "jest -c jest.config.js",
```

Comment:
Since the API package is intended to run in a Node environment, it would be nice to keep these tests around.

Reply:
Got it, the node-tests are back!